py-polars/tests/unit/test_cse.py

```diff
@@ -220,8 +220,8 @@ def test_cse_expr_selection_context(monkeypatch: Any, capfd: Any) -> None:
     assert_frame_equal(result, expected)
 
     out = capfd.readouterr().err
-    assert "run ProjectionExec with 2 CSE" in out
-    assert "run StackExec with 2 CSE" in out
+    # assert "run ProjectionExec with 2 CSE" in out
+    # assert "run StackExec with 2 CSE" in out
```

Comment:
What is the correct way to check this, now that the executor doesn't know it is handling CSE? Just check the stringification of `explain`?

Reply:
Yes, using `explain` and check for CSE is fine. :+1:

Reply:
Done.